parlai/mturk/webapp/dev/app.jsx

```diff
@@ -543,6 +638,679 @@ class RunPanel extends React.Component {
   }
 }
 
+class AssignmentInstructions extends React.Component {
+  render() {
+    let instructions = this.props.data;
```

Comment:
is this similar to swift, where `let` gives you an immutable/constant ref?

Reply:
let is limited in scope to the current block. I should be moving more of my `var`s to be `let`s, but I haven't taken the time to clean yet.